resources/copy-package-json.js

```diff
@@ -11,8 +11,19 @@
  */
 
 const fs = require('fs');
+const assert = require('assert');
 
 const package = require('../package.json');
 delete package.scripts;
 delete package.devDependencies;
+
+const { version } = package;
+const match = /^[0-9]+\.[0-9]+\.[0-9]+-?([^.]*)/.exec(version);
+assert(match, 'Version does not match semver spec.');
+const tag = match[1];
+assert(!tag || tag !== 'rc', 'Only "rc" tag is supported.');
```

Comment:
Shouldn't this be `assert(!tag || tag === 'rc', 'Only "rc" tag is supported.');`? We only support the RC tag, so we don't want to allow non-`rc` tags getting through.

Reply:
@mjmahone Thanks. Fixed